metrics/src/counter.rs

```diff
@@ -162,6 +167,8 @@ mod tests {
 
     #[test]
     fn test_counter() {
+        env::set_var("RUST_LOG", "info");
```

Comment:
Danger:  `env::set_var()` can cause test concurrency issues.   Imagine `cargo test` running two tests that `env::set_var()` the same variable

Reply:
hmmm, that's true. What's the best way to set RUST_LOG for the test in CI (as this requires log::Level::Info)?

Reply:
Yeah IDK.  The logging crates by design use a global configuration.  Not sure they can be overridden by a specific thread.  

Something like this is perhaps slightly better,
https://github.com/solana-labs/solana/blob/e920191de0eb781dcd398b74fe6aa84e596f9dc0/gossip/src/main.rs#L23, because it doesn't muck with the environment and only sets the logging level if it's not already set.